modules/installing-rhacs-ocp-steps.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-high-level-overview
+:_mod-docs-content-type: CONCEPT
+[id="installing-rhacs-ocp-steps_{context}"]
+= Installation steps for RHACS on OpenShift Container Platform
+
+[id="installing-ocp-operator-steps"]
+== Installing {product-title-short} on {osp} by using the {product-title-short} Operator
+
+. On the {osp} cluster, install the {product-title-short} Operator into the `rhacs-operator` project, or namespace.
+. On the {osp} cluster that will contain Central, called the central cluster, use the {product-title-short} Operator to install Central services into the `stackrox` project. One central cluster can secure multiple clusters.
```

Comment:
s/central cluster/Central cluster ??

In other places in the PR, [for example](central cluster), Central is capitalized.